.travis.yml

```diff
@@ -59,6 +59,7 @@ before_install:
   - echo "test --test_output=errors" >>~/.bazelrc
 
 install:
+  - pip install flake8
```

Comment:
This should be pinned to a specific version (the latest one, presumably) like the pip install commands below - this ensures that our travis runs are as reproducible as possible.